docker/vpp-vswitch/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# compile vpp-agent needs vpp libs
+FROM contiv/vpp-base-builder:b59bd659 as builder
+
+ENV GOLANG_VERSION 1.9.3
+ENV GOPATH /root/go
+ENV PATH $PATH:/usr/local/go/bin:$GOPATH/bin
+
+RUN apt-get update \
+    && apt-get install -y curl git make \
+    && rm -rf /var/lib/apt/lists/* \
+    && curl -L "https://golang.org/dl/go$GOLANG_VERSION.linux-amd64.tar.gz" | tar -C /usr/local -xzf - \
+    && go get -u github.com/golang/lint/golint
+
+COPY . $GOPATH/src/github.com/contiv/vpp
+
+# lint and only build vpp-agent
+RUN cd $GOPATH/src/github.com/contiv/vpp \
+    && make lint agent
+
+FROM contiv/vpp-base:b59bd659
+
+RUN apt-get update \
+    && apt-get install -y ethtool supervisor
```

Comment:
Perhaps you could remove the apt lists here to make the image a bit smaller?

Reply:
done